### DISCUSSION TITLE
[Feature Request] Dynamic service name based on var in trace datasource

### DISCUSSION THREAD

**Discussion by purwandi** [grafana/grafana#54335] [Dashboards, Panels & Visualizations]
# Feature requested / solution

If would be great if we have ability to passing dynamic service name into trace panel.

<img width="719" alt="Screen Shot 2022-08-26 at 21 49 48" src="https://user-images.githubusercontent.com/1058672/186934099-4a903e15-b23e-409c-8b84-3776d22e757c.png">
 
# Problem

I want to visualize trace data into graph panel and make filtering service name, but unfortunately service name is can't be passed by variable.

**Comment by purwandi** [comment]
Never mind, actually we can pass the variable to :-)

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
